src/components/App.jsx

```diff
@@ -22,7 +23,8 @@ Form.addInputTypes({
   email: GSTextField,
   script: GSScriptField,
   select: GSSelectField,
-  password: GSPasswordField
+  password: GSPasswordField,
+  autocomplete: GSAutoComplete
```

Comment:
Is it possible to do this without including it in input types here?  (instead, using it by directly importing `GSAutoComplete`)?

My understanding is that addInputTypes is deprecated in newer React-formal versions -- so the others here will have to be changed the same way, but it would be good not to add to the migration burden.

See https://github.com/jquense/react-formal/issues/167

Reply:
@lperson pointed out to me that until we upgrade react-formal, we can't do it any other way -- so it's chicken/egg, and we shouldn't block on this.

Reply:
In light of this, would you be so kind as to approve 🙏